lblod/app-digitaal-loket
# Add queries to remove organizations

## Ticket ID

OP-3124 + OP-3125

## PR Description

> ⚠️ DO NOT MERGE INTO DEV OR MASTER!
>
> Once approved, it has to go into a hotfix release as long as the LMB cutover has not been performed yet.

This PR removes unneeded organizations; it is similar to https://github.com/lblod/app-organization-portal/pull/451.

## How to Test

If possible, try to run a production backup, run the migrations and inspect the important URIs to see if they have successfully deleted (e.g., bestuurseenheid URI, person + account URIs, conversations + messages + file URIs).
